### PR TITLE
Adjust to new perltidy in os-autoinst#563fb45

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
Needs https://github.com/os-autoinst/os-autoinst/pull/662

File: Makefile [hunk 1]
```diff
@@ -23,6 +23,7 @@ testing" && exit 2)
 tools/tidy: os-autoinst/
 	@test -e tools/tidy || ln -s ../os-autoinst/tools/tidy tools/
 	@test -e tools/absolutize || ln -s ../os-autoinst/tools/absolutize tools/
+	@test -e .perltidyrc || ln -s ../os-autoinst/.perltidyrc ./
```
Comment (coolo) [inline Makefile]: submodules don't follow HEAD

**Comment by okurz** [comment]
rebased and checked. In this PR only the consistent `qw()` usage is included as the addition of the new .perltidyrc would involve many more changes and I don't yet know how we could handle it. We have just too many too long lines.

**Comment by okurz** [comment]
wait, that was somehow useless. https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2182 is for 'qw()' only. I will start there again.

**Comment by okurz** [comment]
Still in test, verified working locally. Please expedite review so that others can rebase and fix it.

**Comment by coolo** [comment]
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2239 - the link has to be without ..

**Comment by okurz** [comment]
updated

**Pull request merged**